components/gitpod-cli/cmd/git-commit-message-helper.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2025 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/gitpod-io/gitpod/gitpod-cli/pkg/gitpod"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var gitCommitMessageHelperOpts struct {
+	CommitMessageFile string
+}
+
+var gitCommitMessageHelper = &cobra.Command{
+	Use:    "git-commit-message-helper",
+	Short:  "Gitpod's Git commit message helper",
+	Long:   "Automatically adds Tool information to Git commit messages",
+	Args:   cobra.ExactArgs(0),
+	Hidden: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		ctx, cancel := context.WithTimeout(cmd.Context(), 5*time.Second)
+		defer cancel()
+
+		wsInfo, err := gitpod.GetWSInfo(ctx)
+		if err != nil {
+			return err
```

Comment:
Will not 0 exit code affect / block user commits? If yes, we should not return err

Reply:
Good point, let's remove the non-zero returns